model/src/w3swldmd.F90

```diff
@@ -198,6 +222,26 @@ SUBROUTINE W3SWL4 (A, CG, WN, DAIR, S, D)
 !/
       END SUBROUTINE W3SWL4
 !/ ------------------------------------------------------------------- /
+!
+!> @author H. L. Tolman @date 16-Feb-2012
```

Comment:
We usually put the brief description at the top, and the author/date at the bottom of the documentation block.

Reply:
Sounds good, we can follow that convention.  I'll make the change.